airflow/config_templates/config.yml

```
@@ -241,11 +241,20 @@ core:
     allowed_deserialization_classes:
       description: |
         What classes can be imported during deserialization. This is a multi line value.
-        The individual items will be parsed as regexp. Python built-in classes (like dict)
-        are always allowed. Bare "." will be replaced so you can set airflow.* .
+        The individual items will be parsed as a pattern to a glob function. 
+        Python built-in classes (like dict) are always allowed. 
       version_added: 2.5.0
       type: string
-      default: 'airflow\..*'
+      default: 'airflow.*'
+      example: ~
+    allowed_deserialization_classes_regexp:
+      description: |
+        What classes can be imported during deserialization. This is a multi line value.
+        The individual items will be parsed as regexp patterns. 
+        This is a secondary option to ``allowed_deserialization_classes``.
+      version_added: 2.7.3
```

2.7.3 already released
```suggestion
      version_added: 2.8.1
```